src/main/jbake/content/advanced/index.adoc

```diff
@@ -1,11 +0,0 @@
-= Advanced
```

Comment:
Also the same. I think that the site is pretty new and if linked then it's an easy change.